README.md

```diff
@@ -64,32 +64,56 @@ you need to add PYTHONUNBUFFERED=1;PYDEVD_USE_FRAME_EVAL=NO to your Run/Debug en
     - **files:**
       [TheBloke/Mistral-7B-Instruct-v0.2-GGUF/tree/main](https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.2-GGUF/tree/main)
 
-## Run with docker (** NOT IMPLEMENTED **)
+## Run with docker
 
-- **CPU Mode**
+- **CPU Mode**  (** NOT IMPLEMENTED **)
     - CPU - slow but reliable
         - ```
-            docker run --name alexandria-project alexandria-project -v path/to/model:/usr/model --env-file https://raw.githubusercontent.com/alexandriaproject-io/llm-inference/main/env-samples/.env.cpu.example 
+            docker build -f docker/Dockerfile.cuda12 -t alexandria_project .
+
+            docker run -p 5050:5050 --env-file ./env-samples/.env.auto.example \
+        -v $(pwd)/TinyLlama-1.1B-Chat-v1.0:/usr/src/app/app/TinyLlama-1.1B-Chat-v1.0 \
+        -e MODEL_PATH=TinyLlama/TinyLlama-1.1B-Chat-v1.0 \
+        --name alexandria_projecdt_instance \
+        alexandria_project python main.py --multiprocess
           ```
 
 - **CUDA Mode**
+
+    - requirements:
+      - nvidia-drivers (instruction in the sections above according to your os)
+      - [cuda](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64)
+        
     - Verify docker access:
         - ```
             docker run --gpus all nvcr.io/nvidia/k8s/cuda-sample:nbody nbody -gpu -benchmark
             ```
-            - If not please
-              install https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html
-
+            - If not install [docker-nvida2](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)
 
 - **Cuda 12.1 or later ( GTX 20xx, 30xx, 40xx )**
     - ```
-      docker run --gpus all --name alexandria-project alexandria-project -v path/to/model:/usr/model --env-file https://raw.githubusercontent.com/alexandriaproject-io/llm-inference/main/env-samples/.env.cuda.example 
+       docker build -f docker/Dockerfile.cuda12 -t alexandria-project .  
```

Comment:
Remove the docker build part, its unnecessary

Reply:
I don't understand why it is unnecessary, I suggest leaving the building part because it is different for each type of installation.

Reply:
people who want to build will know how to run docker build -f dockerfile, we also dont want them using our tags...

Reply:
removed